core/transaction_pool.go

```diff
@@ -109,7 +114,17 @@ func (pool *TxPool) eventLoop() {
 }
 
 func (pool *TxPool) resetState() {
-	pool.pendingState = state.ManageState(pool.currentState())
+	currentState, err := pool.currentState()
+	if err != nil {
+		glog.V(logger.Info).Infoln("Transaction pool: failed to get current state: %v", err)
+		return
+	}
+	managedState := state.ManageState(currentState)
+	if err != nil {
+		glog.V(logger.Info).Infoln("Transaction pool: failed to get managed state: %v", err)
```

Comment:
btw, the "Transaction pool: " prefix doesn't add much value because the log will say

```
timstamp  pid   txpool.go:124] message
```